src/extension/tab/helpers.ts

```diff
@@ -23,11 +29,37 @@ export type QueryOptions = Pick<
   | "notifyOnNetworkStatusChange"
 > & { nextFetchPolicy?: string };
 
+export interface SerializedApolloError extends Pick<ApolloError, "message"> {
+  name: "ApolloError";
+  clientErrors: string[];
+  networkError?: SerializedError;
+  graphQLErrors: Array<
+    Pick<GraphQLFormattedError, "message" | "path" | "extensions">
+  >;
+  protocolErrors: string[];
+}
+
+export interface SerializedError {
+  message: string;
+  name: string;
+  stack?: string;
+}
+
 export type QueryInfo = {
```

Comment:
Could we rename this to `QueryDetails` to avoid confusion?

Reply:
Done! I went ahead and renamed `MutationInfo` as well to keep the same convention: [d1f2cae](https://github.com/apollographql/apollo-client-devtools/pull/1429/commits/d1f2cae0f7a72be5fe53bbc36c65008aba1442fe)